src/shapes/Object/StackedObject.ts

```diff
@@ -42,17 +41,14 @@ export class StackedObject<
    * A reference to the parent of the object
    * Used to keep the original parent ref when the object has been added to an ActiveSelection, hence loosing the `group` ref
    */
-  declare __owningGroup?: Group;
+  declare parent?: Group;
 
   /**
    * Returns instance's parent **EXCLUDING** `ActiveSelection`
    * @param {boolean} [strict] exclude canvas as well
    */
   getParent<T extends boolean>(strict?: T): TAncestor | undefined {
```

Comment:
remove this method because the dev can access `group` or `parent` or check equality